cpp/rocksdb4pl.cpp

```diff
@@ -1380,12 +1377,8 @@ static bool
 unify_enum_key(PlTerm t, const enum_state& state)
 { if ( state.type == ENUM_PREFIX )
   { rocksdb::Slice k(state.it->key());
-
-    if ( k.size_ >= state.prefix.length() &&
-	 memcmp(k.data_, state.prefix.data(), state.prefix.length()) == 0 )
-    { k.data_ += state.prefix.length();
-      k.size_ -= state.prefix.length();
-
+    if ( k.starts_with(state.prefix) )
```

Comment:
I think that this code is equivalent but please check against
https://github.com/facebook/rocksdb/blob/981fd432fa2441fc10a59a462bd14906ccb1c0e0/include/rocksdb/slice.h#L112
https://github.com/facebook/rocksdb/blob/981fd432fa2441fc10a59a462bd14906ccb1c0e0/include/rocksdb/slice.h#L78